supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -69,6 +69,17 @@
 
 *See also*: xref:management-cli[management CLI]
 
+[discrete]
+[[natively-compiled-code]]
+==== image:images/yes.png[yes] natively compiled code (noun)
+*Description*: _Natively compiled code_ is code written in a programming language that compiles to machine code with a resulting binary executable file. Such code can be run stand-alone.
```

Comment:
Remove hyphen (per our SSG guidance). Though it does kind of sound off at the end of a sentence like that, there might be a better way to state it.

```suggestion
*Description*: _Natively compiled code_ is code written in a programming language that compiles to machine code with a resulting binary executable file. Such code can be run standalone.
```